styles/styles.css

```diff
@@ -0,0 +1,370 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700;900&family=Roboto:wght@300;400;500;700&display=swap');
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    border: 0;
+}
+
+:root {
+    --wit: #fff;
+    --paars: #5722CD;
+    --blauw: #4845E4;
+    --rose: #ED4385;
+    --zwart: #0C0C0C;
+    --lichtpaars: #F2F0FF;
+    --lichtgrijs: #F3F6F9;
+    --grijs: #7A7A7A;
```

Comment:
Ik ben misschien een zeurpiet, maar het is wel goed om te leren denk ik. Probeer je code (en dus ook css) waar mogelijk in het Engels te formuleren 😄

Reply:
Thanks voor het nakijken!